api/server.go

```diff
@@ -77,8 +83,80 @@ func (s *Server) RatsdChares(w http.ResponseWriter, r *http.Request, param Ratsd
 		return
 	}
 
+	nonce, err := base64.RawURLEncoding.DecodeString(requestData.Nonce)
+	if err != nil {
+		errMsg := fmt.Sprintf("fail to decode nonce from the request: %s", err.Error())
+		p := &problems.DefaultProblem{
+			Type:   string(TagGithubCom2024VeraisonratsdErrorInvalidrequest),
+			Title:  string(InvalidRequest),
+			Detail: errMsg,
+			Status: http.StatusBadRequest,
+		}
+		s.reportProblem(w, p)
+		return
+	}
 	s.logger.Info("request nonce: ", requestData.Nonce)
+	s.logger.Info("request media type: ", *(param.Accept))
+
+	// Use a map until we finalize ratsd output format
+	eat := make(map[string]interface{})
+	collection := cmw.NewCollection("tag:ietf.org,2025:ratsd")
```

Comment:
```suggestion
	collection := cmw.NewCollection("tag:github.com,2025:veraison/ratsd/cmw")
```

Reply:
Thanks for pointing out the better use of `cmwc_t`